src/OrchardCore/OrchardCore.Mvc.Core/SharedViewCompilerProvider.cs

```diff
@@ -93,6 +95,44 @@ private IViewCompiler CreateCompiler()
                 provider.PopulateFeature(assemblyParts, feature);
             }
 
+            var featureProvider = featureProviders.OfType<ViewsFeatureProvider>().FirstOrDefault();
+
+            if (!_hostingEnvironment.IsDevelopment() && featureProvider != null)
```

Comment:
If this code was copied from somewhere, please reference the github link to it, for future changes

Reply:
Only one part of the code, i will retrieve this part and reference it.